doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+
++--------------+---------------------------------------------------------------+
+| Plutus use case examples                                                     |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-use-cases``                              |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-ledger, plutus-ledger-api, plutus-ledger-constraints,  |
+|              | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | Maintained. The examples are used as test cases.              |
+|              | While they work in the plutus-contract emulator, they         |
+|              | are not guaranteed to work on the actual Cardano network.     | 
++--------------+---------------------------------------------------------------+
+
+12. PAB (Plutus application backend)
+------------------------------------------
+
+`PAB <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_ 
+is a web server library for managing the state of Plutus contract instances. 
+The PAB executes the off-chain component of Plutus applications. It manages 
+application requests to the wallet backend, the Cardano node and the chain-index. 
+PAB stores the application state and offers an HTTP REST API for managing application 
+instances. 
+
+PAB wraps the contracts built with plutus-contract. It is the central point of 
+contact for everything, integrating all the Cardano components. 
+
++--------------+---------------------------------------------------------------+
+| PAB                                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-pab``                                    |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-contract, plutus-contract emulator,                    |
+|              | plutus-chain-index-core                                       |
++--------------+---------------------------------------------------------------+
+| State        | Maintenance mode for bugs, but not a high priority component. |
+|              | We are not adding any new features unless they are            |
+|              | contributed by external stakeholders.                         | 
++--------------+---------------------------------------------------------------+
+
+13. PAB simulator 
+---------------------------
+
+`PAB simulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_ 
+is a simulator for the PAB that is used for testing a plutus application in a 
+simulated environment. 
+
```

Comment:
We need more details. How is this different from the emulator?

Reply:
I'm not sure I agree this is not a high priority component. It may be important when we roll out the new testing framework. @raduom what do you think?